api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -9,6 +9,25 @@ import "google/protobuf/wrappers.proto";
 
 option go_package = "github.com/illumio/cloud-operator/api/illumio/cloud/k8sclustersync/v1";
 
+// Application-level keepalives. While gRPC already has a system-level keepalive
```

Comment:
```suggestion
// Application-level keepalives. While gRPC already has an HTTP-level keepalive
```

You may also refer to the gRPC spec:
https://github.com/grpc/grpc/blob/master/doc/keepalive.md
https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#ping-frame